app/models/concerns/authorizable.rb

```diff
@@ -6,6 +6,11 @@
   manage_adopter_foster_profiles
 ].freeze
 
+FOSTERER_PERMISSIONS = %i[
+  create_adopter_foster_profiles
```

Comment:
can we also add `view_adopter_foster_dashboard` here, too? I just merged that in.